helm/resource_helm_release_test.go

```diff
@@ -64,16 +66,51 @@ func TestAccResourceRelease_basic(t *testing.T) {
 		},
 	})
 }
+
+func TestAccResourceRelease_set_wo(t *testing.T) {
+	name := randName("writeonly")
+	namespace := createRandomNamespace(t)
+	defer deleteNamespace(t, namespace)
+
+	resource.Test(t, resource.TestCase{
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			// FIXME use const from terraform-plugin-testing once v1.11.0 is released.
+			tfversion.SkipBelow(version.Must(version.NewVersion("1.11.0"))),
+		},
+		// PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV6ProviderFactories: protoV6ProviderFactories(),
+		// CheckDestroy:             testAccCheckHelmReleaseDestroy(namespace),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccHelmReleaseConfig_set_wo(testResourceName, namespace, name, "1.2.3"),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("helm_release.test", "status", release.StatusDeployed.String()),
+					resource.TestCheckResourceAttr("helm_release.test", "set_wo_revision", "1"),
+					resource.TestCheckResourceAttr("helm_release.test", "metadata.values", "{}"),
+				),
+			},
+			{
+				Config: testAccHelmReleaseConfig_set_wo2(testResourceName, namespace, name, "1.2.3"),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("helm_release.test", "status", release.StatusDeployed.String()),
+					resource.TestCheckResourceAttr("helm_release.test", "set_wo_revision", "2"),
+					resource.TestCheckResourceAttr("helm_release.test", "metadata.values", "{}"),
+				),
+			},
+		},
+	})
+}
+
 func TestAccResourceRelease_emptyVersion(t *testing.T) {
 	name := randName("basic")
 	namespace := createRandomNamespace(t)
 	defer deleteNamespace(t, namespace)
 
 	resourceName := "helm_release.test"
 	resource.Test(t, resource.TestCase{
-		//PreCheck:                 func() { testAccPreCheck(t) },
+		// PreCheck:                 func() { testAccPreCheck(t) },
```

Comment:
We can remove all these comments leftover from the migration in another PR – this is just formatter noise.